conda_forge_tick/auto_tick.py

```diff
@@ -960,6 +955,7 @@ def _is_migrator_done(_mg_start, good_prs, time_per, pr_limit, tried_prs):
 
 def _run_migrator(migrator, mctx, temp, time_per, git_backend: GitPlatformBackend):
     _mg_start = time.time()
+    home_directory = os.getcwd()
```

Comment:
Why call this home_directory if it's not? Should it just be `working_dir` or similar?

Reply:
This came from the original name `BOT_HOME_DIR`. I renamed it to `initial_working_dir` to make it less ambiguous.